tests/pytests/test_scf_options.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+import psi4
+
+from .utils import compare_values
+
+@pytest.mark.parametrize("inp", [
+    pytest.param({'options': {"guess": "core"}}, id="core"),
+    pytest.param({'options': {"guess": "gwh"}}, id="gwh"),
+    pytest.param({'options': {"guess": "huckel"}}, id="huckel"),
+    pytest.param({'late_options': {"guess": "read"}}, id="huckel"),
+    pytest.param({'options': {"guess": "sad"}}, id="sad"),
+    pytest.param({'options': {"guess": "sadno"}}, id="sadno"),
+    pytest.param({'options': {"guess": "sap"}}, id="sap"),
+    ]
+)
+def test_guess_mix_for_broken_symmetry(inp):
+
+    refENuc  =  0.17639240356
+    refSCF   = -0.82648407827446 
+    refBSSCF = -0.99872135103903
+
+    h2 = psi4.geometry("""
+        0 1
+        H
+        H 1 3.0
+        symmetry c1
+        """)
+
+    psi4.set_options({"reference": "uhf", "e_convergence": 12, "basis": "cc-pvdz"})
+    psi4.set_options(inp.get("options", {}))
+
+    thisSCF = psi4.energy("scf")
+    psi4.set_options(inp.get("options_late", {}))
```

Comment:
@JonathonMisiewicz shouldn't this be `'late_options'` as on line 11 instead of `options_late`?

Reply:
Good call - they should be consistent.

Reply:
Fixed